TOC.md

```diff
@@ -111,9 +111,10 @@
     + [Coprocessor Cache](/coprocessor-cache.md)
   + SQL Tuning
     + [Overview](/sql-tuning-overview.md)
-    + [SQL Tuning with `EXPLAIN`](/query-execution-plan.md)
-    + SQL Optimization
-      + [SQL Optimization Process](/sql-optimization-concepts.md)
+    + Understanding the Query Execution Plan
+      + [Overview](/explain-overview.md)
```

Comment:
- Since we don't have multiple topics about query execution plan now, how about keeping one line?
- Is it a must to change the file name from `query-execution-plan.md` to `explain-overview.md`? Because we'll try to avoid changing file names unless it's really necessary, to make it easy to maintain documentation (Let's avoid using too many aliases and try to keep the URL stable if possible).
```suggestion
    + [Understanding the Query Execution Plan](/explain-overview.md)
```

Reply:
I would prefer to keep it as is. I have PRs for the added content being reviewed in https://github.com/pingcap/docs/pull/3862 and https://github.com/pingcap/docs/pull/3861 - and will add a third one later today.

Reply:
By "keep it as is", I think you mean keeping your changes, right? And the newly added content in two other PRs will be put under "Understanding the Query Execution Plan"?

Reply:
That is correct. The TOC I proposed to the SQL team is as follows: